CoinControl/Models/AccountType.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  AccountType.swift
+//  CoinControl
+//
+//  Created by Md. Asadul Islam on 15/2/24.
+//
+
+import Foundation
+//MARK: sshs
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>warning: (blankLinesAroundMark) Insert blank line before and after MARK: comments.